cognite/client/data_classes/transformations/common.py

```diff
@@ -163,6 +188,37 @@ def __hash__(self) -> int:
         return hash((self.type, self.model_external_id, self.space_external_id, self.instance_space_external_id))
 
 
+class Instances(TransformationDestination):
+    def __init__(
+        self,
+        view_external_id: str = None,
+        view_version: str = None,
+        view_space_external_id: str = None,
+        instance_space_external_id: str = None,
+    ):
+        warnings.warn(
+            "Feature DataModelStorage is in beta and still in development. "
+            "Breaking changes can happen in between patch versions.",
+            stacklevel=2,
+        )
+        super().__init__(type="instances")
+        self.view_external_id = view_external_id
+        self.view_version = view_version
+        self.view_space_external_id = view_space_external_id
+        self.instance_space_external_id = instance_space_external_id
+
+    def __hash__(self) -> int:
+        return hash(
```

Comment:
Out of curiosity, why are you implementing dunder hash for all transformation data classes?

Reply:
Great question, I don't have much idea about it, they are added long time ago I think. Maybe it's used for encoding/decoding ? We can create a draft PR to remove all the hash methods to see if it breaks smthing? 😂

Reply:
Haha okey, I had a feeling 🤣